IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_OCSRigidSupport/DocPropertySet.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_OCSRigidSupport" UniqueId="37ce8447-2a11-e7c4-7e22-82827dd949f7" ApplicableType="IfcMember/(.MEMBER.)" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="AssemblyInstruction_3W5i6nbkP9NBt_M6PTB5jW" />
+		<DocProperty xsi:nil="true" href="Stagger_1dWD_52DDFheUkLfvK_Wga" />
+	</Properties>
+</DocPropertySet>
+
```

Comment:
Just checking. This Pset has IfcMember/MEMBER as applicable type (not IfcMechanicalFastener). Duplication?

Reply:
I have checked it. We didn't mapped it at the begginning to the good PDT... It was not a Mechanical Fastener but a Member..

Reply:
Need some clarification, these `IfcMember` psets should have been in another PR and you left them here for the sake of simplicity?

Reply:
@fhulin im happy for this to stay here for simplicity one note: Stagger should be `ContactWireStagger_1dWD_52DDFheUkLfvK_Wga` as this was a change to another occurance of the stagger property in: [Pset_DistributionSystemTypeOverheadContactlineSystem](https://github.com/bSI-InfraRoom/IFC-Specification/blob/f722d49608e936bd36636365537c5b4dea5d9ae1/IFC4x3/Sections/Domain%20specific%20data%20schemas/Schemas/IfcRailDomain/PropertySets/Pset_DistributionSystemTypeOverheadContactlineSystem/DocPropertySet.xml)

Reply:
Done. de4bcf9de7f5e2e1d9e24e66e3b338da3b5506eb